### ISSUE TITLE
consider adding notyet.badssl.com (future notBefore)

### ISSUE THREAD

**Issue by jschauma** [chromium/badssl.com#495]
It might be useful to have a test to verify validation of the 'notBefore' date.  I don't know how far in the future CAs let you request a cert, so there would eventually be a renewal needed once the future date becomes present/past.

**Comment by hgc81538** [comment]
I think it is hard to find a CA to issue such cert.

You might change your computer clock to simulate.

**Comment by ldemailly** [comment]
could be a self signed future cert I suppose off self-signed.badssl.com for instance

**Comment by christhompson** [comment]
I don't think we will be able to reasonably acquire such a cert (in an automated fashion via ACME). The workaround of simulating via changing your local computer clock seems pretty reasonable to me, so closing this.
